src/logger/HttpLogger_test.go

```diff
@@ -0,0 +1,75 @@
+package logger
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+//Testing HTTP Logger
+
+func TestCreateInstance(t *testing.T) {
+
+	//Creating a single instance
+	httpLogger := NewHttpLogger()
+	helper := GetTestHelper()
+	assert.NotNil(t, httpLogger)
+	assert.Equal(t, helper.mockAgent, httpLogger.Agent())
+	assert.False(t, httpLogger.Enablable())
+	assert.False(t, httpLogger.Enabled())
+	assert.Nil(t, httpLogger.Queue())
+	assert.Nil(t, httpLogger.Url())
+
+}
+
+func TestCreateMultipleInstances(t *testing.T) {
+	//Creating multiple instances
+	url1 := "https://resurface.io"
+	url2 := "https://whatever.com"
+	helper := GetTestHelper()
+
+	logger1 := NewHttpLogger(url1)
```

Comment:
Go doesn't allow overloading to these constructors will have to have the names of the parameters in their name, specifically here the names will be `NewHttpLoggerUrl`